client/src/app/+videos/+video-watch/shared/comment/video-comments.component.ts

```diff
@@ -145,11 +150,14 @@ export class VideoCommentsComponent implements OnInit, OnChanges, OnDestroy {
     })
   }
 
-  loadMoreThreads () {
+  loadMoreThreads (reset = false) {
     const params = {
       videoId: this.video.uuid,
       videoPassword: this.videoPassword,
-      componentPagination: this.componentPagination,
+      componentPagination: {
+        ...this.componentPagination,
+        currentPage: reset === true ? 1 : this.componentPagination.currentPage
```

Comment:
I think that if `reset === true` you must do `this.componentPagination.currentPage = 1` or you'll loose pagination in future calls